pages_/my-shop.tsx

```diff
@@ -117,6 +131,16 @@ const MyShop = ({ isLoggedIn, shopId }: Props) => {
     }
   }, [isLoggedIn, shopId]);
 
+  useEffect(() => {
+    const res = window.localStorage.getItem(POSTER_NOTIFICATION);
+
+    if (res) {
```

Comment:
shouldn't this be `res === 'true'` just to be sure?

Reply:
it shouldn't be set with another value, but adding === 'true' won't hurt anybody :)